config/src/main/java/org/springframework/security/config/annotation/web/configurers/HeadersConfigurer.java

```diff
@@ -487,12 +491,58 @@ public ReferrerPolicyConfig referrerPolicy(ReferrerPolicy policy) {
 	 * @throws IllegalArgumentException if policyDirectives is {@code null} or empty
 	 * @since 5.1
 	 * @see FeaturePolicyHeaderWriter
+	 * @deprecated Use {@link #permissionsPolicy(Customizer)} instead.
 	 */
+	@Deprecated
 	public FeaturePolicyConfig featurePolicy(String policyDirectives) {
 		this.featurePolicy.writer = new FeaturePolicyHeaderWriter(policyDirectives);
 		return this.featurePolicy;
 	}
 
+	/**
+	 * <p>
+	 * Allows configuration for
+	 * <a href="https://w3c.github.io/webappsec-permissions-policy/">Permissions
+	 * Policy</a>.
+	 * </p>
+	 *
+	 * <p>
+	 * Configuration is provided to the {@link PermissionsPolicyHeaderWriter} which
+	 * support the writing of the header as detailed in the W3C Technical Report:
+	 * </p>
+	 * <ul>
+	 * <li>Permissions-Policy</li>
+	 * </ul>
+	 * @return the {@link PermissionsPolicyConfig} for additional configuration
+	 * @since 5.5
+	 * @see PermissionsPolicyHeaderWriter
+	 */
+	public PermissionsPolicyConfig permissionsPolicy(String policy) {
+		this.permissionsPolicy.writer = new PermissionsPolicyHeaderWriter(policy);
+		return this.permissionsPolicy;
+	}
```

Comment:
I think I confused you with my previous comment. This was correct before.
```suggestion
	public PermissionsPolicyConfig permissionsPolicy() {
		this.permissionsPolicy.writer = new PermissionsPolicyHeaderWriter();
		return this.permissionsPolicy;
	}
```

Reply:
Oh yes, it's my bad... I revert the change and update the tests